lib/matplotlib/tests/test_compare_images.py

```diff
@@ -54,6 +55,7 @@ def test_image_comparison_expect_rms(im1, im2, tol, expect_rms):
     succeed if compare_images succeeds. Otherwise, the test will succeed if
     compare_images fails and returns an RMS error almost equal to this value.
     """
+    monkeypatch.chdir(tmp_path)
```

Comment:
We agree the copying of the baseline image was redundant, so we removed it.

Reply:
You should also update the code comment so it describes the new approach.  Otherwise the next person to try to change this code will be very confused!

Reply:
We fixed the comments.